library-front/src/components/pages/HomePage/HomePage.tsx

```diff
@@ -0,0 +1,24 @@
+import { FC } from 'react'
+import BookCard from '../../BookCard/BookCard'
+import Footer from '../../Footer/Footer'
+import Header from '../../Header/Header'
+import Navbar from '../../Navbar/Navbar'
+import './HomePage.css'
+
+interface HomePageProps {}
+
+const HomePage: FC<HomePageProps> = () => (
+  <div className='HomePage'>
+    <Header />
+    <Navbar />
+    <div className='home-container'>
```

Comment:
Please be consistent when giving class names. Consistency in class naming is essential for the maintainability and readability of code. It is recommended to choose a naming convention and stick with it throughout the project. it's important to choose a consistent approach and use it consistently.